pkg/diff/README.md

```diff
@@ -16,14 +16,9 @@ type TableDiff struct {
 	// target table
 	TargetTable *TableInstance
 
-	// columns be ignored, will not check this column's data, 
-	// but may use these columns as split field or order by key.
+	// columns be ignored, will not check this column's data
 	IgnoreColumns []string
 
-	// columns be removed, will remove these columns from table info,
-	// and will not check these columns' data, will not use these columns as split field or order by key too.
-	RemoveColumns []string
```

Comment:
only when column's value is different in TiDB and MySQL need set ignore-column. If data is different and use this column to split chunk, the chunk's range in TiDB and MySQL is also different. so we don't need both have ignore-column and remove-column, I delete the remove-column, and use ignore-column to replace remove-column.